tools/regenerate-lockfile-fixtures.ts

```diff
@@ -49,11 +51,22 @@ async function initForgeProject(dir: string, template: string): Promise<void> {
   }
 
   return new Promise((resolve, reject) => {
-    const child = spawn('node', [cliPath, '.', '--template', template], {
-      cwd: dir,
-      stdio: 'inherit',
-      shell: process.platform === 'win32',
-    });
+    const child = spawn(
```

Comment:
TODO: we should make this use the API directly instead of wiring it through the CLI.